src/containers/Experiment/index.js

```diff
@@ -23,6 +23,12 @@ import ScrollTopOnMount from '../../components/ScrollTopOnMount';
 import Anchor from '../../components/Anchor';
 import Technology from './Technology';
 
+const DataBaseNames = {
```

Comment:
It probably makes more sense to use `DatabaseNames`, because `database` is one word, not two.